File: SchoolData/src/main/java/org/gb/homework/schooldata/view/AssignView.java
package org.gb.homework.schooldata.view;

import org.gb.homework.schooldata.model.Student;
import org.gb.homework.schooldata.model.Teacher;

import java.util.List;

public class AssignView {
    public void showTeacher(Teacher teacher) {
        System.out.printf("\nУчитель - %d %s\n", teacher.getId(), teacher.getName());

        List<Student> group = teacher.getGroup();

        if (group.size() == 0)
            System.out.println("\tстуденты не назначены");

        System.out.printf("\tстуденты:");
        for (var s : group) {
            System.out.printf("\t%d %s |", s.getId(), s.getName());
        }
        System.out.println();
    }

    public void showTitle() {
        System.out.println("\nРаспределение студентов по учителям:");
    }
}

File: csharp/SchoolData/View/AssignView.cs
using Gb.Homework.SchoolData.Model;

namespace Gb.Homework.SchoolData.View
{


    public class AssignView
    {
        public void ShowTeacher(Teacher teacher)
        {
            Console.WriteLine($"\nУчитель - {teacher.Id} {teacher.Name}");

            List<Student> group = teacher.getGroup();

            if (group.Count == 0)
                Console.WriteLine("\tстуденты не назначены");

            Console.WriteLine("\tстуденты:");

            foreach (Student student in group)
            {
                Console.Write($"\t{student.Id} {student.Name} |");
            }
            Console.WriteLine();
        }

        public void ShowTitle()
        {
            Console.WriteLine("\nРаспределение студентов по учителям:");
        }
    }

}
